### ISSUE TITLE
Options do not show for selectized multiple selects after the first control (v0.12.0)

### ISSUE THREAD

**Issue by xuanwu** [selectize/selectize.js#744]
I have two selects with the multiple attribute, and I try calling selectize on both of them at the same time using $('select').selectize().  When using v0.12.0, the first one shows the correct options.  The second one doesn't show any options at all.

Here is the jsfiddle using selectize v0.12.0:
http://jsfiddle.net/rpsepbed/1/

Here is the jsfiddle using selectize v0.11.2:
http://jsfiddle.net/rpsepbed/2/

The only thing I changed was the version of selectize.js.  Seems like pretty big breakage; I'm surprised I couldn't find it in the open issues.

(The reason I want to use v0.12.0 is for its selectOnTab option.)

**Comment by gmcinalli** [comment]
I've the same problem! Bug?

**Comment by ghost** [comment]
I've the same problem with v0.12.0 

I was forced to change the script

```
$('select').selectize();
```

to

```
$('select').each(function(){
    $(this).selectize()
});
```

other solution ?

**Comment by xuanwu** [comment]
@bonv I like your workaround!

**Comment by xuanwu** [comment]
@gmcinalli I think so.  I hope they fix it in the next point release.  I've got multiple places in my project where I'm calling selectize with various configurations...

**Comment by leevigraham** [comment]
I've had the same issue. Implemented the same work around as @bonv: https://github.com/brianreavis/selectize.js/issues/744#issuecomment-84962500

**Comment by Preen** [comment]
Me2. Spent 2 hours...

**Comment by brianreavis** [comment]
This is fixed in [0.12.1](https://github.com/brianreavis/selectize.js/releases/tag/v0.12.1)
